src/components/ProfileEditPage.js

```diff
@@ -28,32 +59,41 @@ const ProfileEditPage = () => {
     navigate(`/profile`)
   }
   return (
-    <div>
-      <div>
-        <form onSubmit={createEmail} onReset={createReset}>
-          <div>
-            <label htmlFor="emailInput">
-              <TextField
-                id="emailInput"
-                type="text"
-                variant="outlined"
-                margin="normal"
-                required
-                fullWidth
-                // inputRef={inputRef}
-                //id="email"
-                label="Email"
-                name="email"
-                placeholder="unicorn@usepow.app"
-                // value={state.email}
-                autoComplete="email"
-                // onChange={handleChange("email")}
-                //  InputLabelProps={{ shrink: true }}
-              />
-            </label>
-          </div>
+    <div className={classes.root}>
+      <form
+        onSubmit={createEmail}
+        onReset={createReset}
+        className={classes.form}
+      >
+        <Toolbar />
```

Comment:
This` <toolbar /> `is hidden under 
the` <toolbar /> `in line 96
and the` <AppBar /> `in line 90?